### ISSUE TITLE
Line will not parse due to incorrect `Crowding` model schema

### ISSUE THREAD

**Issue by lmmx** [lmmx/tubeulator#52]
This looks like a schema bug, perhaps it should be fixed upstream of running the codegen

Breakpointing after commenting out the `crowding` key in the `Line` model of the Line API

```py
>>> import tubeulator
>>> lines = tubeulator.fetch.line.lines_by_modes("tube")
> /home/louis/dev/tubeulator/src/tubeulator/api/request.py(548)parse()
-> return parsed
(Pdb) import json
(Pdb) pp json.loads(response.content)[0]
{'$type': 'Tfl.Api.Presentation.Entities.Line, Tfl.Api.Presentation.Entities',
 'created': '2024-05-14T14:35:06.647Z',
 'crowding': {'$type': 'Tfl.Api.Presentation.Entities.Crowding, '
                       'Tfl.Api.Presentation.Entities'},
 'disruptions': [],
 'id': 'bakerloo',
 'lineStatuses': [],
 'modeName': 'tube',
 'modified': '2024-05-14T14:35:06.647Z',
 'name': 'Bakerloo',
 'routeSections': [],
 'serviceTypes': [{'$type': 'Tfl.Api.Presentation.Entities.LineServiceTypeInfo, '
                            'Tfl.Api.Presentation.Entities',
                   'name': 'Regular',
                   'uri': '/Line/Route?ids=Bakerloo&serviceTypes=Regular'}]}
```

It looks to me like this is in fact not a bug so much as the keys are optional (maybe to indicate crowding info is not being provided but might be).

I suspect the schema is wrong here (the fields of the Crowding model should be indicated to be optional)

The Crowding model is:

```py
class Crowding(BaseModel):
    """
    Autogenerated from Line::Tfl.Api.Presentation.Entities.Crowding
    """

    model_config = ConfigDict(
        alias_generator=AliasGenerator(validation_alias=to_camel_case),
    )

    PassengerFlows: list["PassengerFlowModel"]
    TrainLoadings: list["TrainLoadingModel"]
    _source_schema_name: str = PrivateAttr(default='Line')
    _component_schema_name: str = PrivateAttr(default='Tfl-5')
```

This would be fine if the list fields were given as `= []` (i.e. optional)

**Comment by lmmx** [comment]
There may be more of these (this may not be unique), which indicates **we need tests**

**Comment by lmmx** [comment]
The Crowding schema isn't the problem exactly it's the default for non required array properties, it should be set to `= []`

https://github.com/lmmx/tubeulator/blob/18b4c5cffce1d313e199a8d30632ed72412556ce/src/tubeulator/codegen/gen/dto.py#L274

**Comment by lmmx** [comment]
Fixed and tested in #58
